updates/next.json

```diff
@@ -1,16 +1,24 @@
 {
   "stream": "next",
   "metadata": {
-    "last-modified": "2020-06-22T18:38:34Z"
+    "last-modified": "2020-06-25T15:18:18Z"
   },
   "releases": [
     {
-      "version": "32.20200615.1.2",
+      "version": "32.20200615.1.3",
+      "metadata": {
+        "barrier": {
```

Comment:
Hmm, shouldn't this barrier be in the same array element as the rollout?

Reply:
@lucab - can you say?

Reply:
Yes @jlebon is right, versions are unique keys, see https://github.com/coreos/fedora-coreos-tracker/blob/master/metadata/updates/specifications.md.

Reply:
ok I think I fixed it